doc/.pre_processing.py

```diff
@@ -288,8 +308,20 @@ class Colors:
   changelogfile.write('Changelog\n')
   changelogfile.write('=========\n')
   actualVersion = 0.0
+  experimentalheader = False
   
   for change in changelog:
+    if change[1] == 3:
+      if not experimentalheader:
+        experimentalheader = True
+        changelogfile.write('\n.. list-table:: Experimental (functions are subject to change)\n')
```

Comment:
changelogfile.write('\n.. list-table:: Experimental (functions are subject to change)')
changelogfile.write('\n  :widths: ' + str(first_column) + ' ' + str(100 - first_column))
changelogfile.write('\n')